lwcomponent/component.go

```diff
@@ -603,3 +617,71 @@ func (c Component) EnterDevelopmentMode() error {
 
 	return nil
 }
+
+// ByVersion type to sort an array of versions by their semver comparison
+type ByVersion []semver.Version
+
+func (a ByVersion) Len() int           { return len(a) }
+func (a ByVersion) Less(i, j int) bool { return a[i].LessThan(&a[j]) }
+func (a ByVersion) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
```

Comment:
Is this needed for sorting. Is it not possible to do this for the packages github readme?
https://github.com/Masterminds/semver#sorting-semantic-versions

Reply:
I agree with Darren, this is exactly whats implemented in the `semver.Collection` type. Can we use it instead?

- https://github.com/Masterminds/semver/blob/master/collection.go

Reply:
Thanks, that's a good point! I've switched to using that, which required making the array into an array of pointers instead of values to make the types work. I think this is okay, and the unit tests should've caught if this caused issues, but please do check this. (This is my first non-trivial PR in Go, so I'm somewhat lacking in the details of the semantics of the language.)